### PR TITLE
[solution]: zh/medium-endswith

### PR DESCRIPTION
**Pull request by likui628**
# Challenge [zh/medium-endswith]

## Checklist

- [x] Make sure that you are opening the PR with a single solution (one solution per PR)
- [x] The solution must have at least several references for the readers to dive in depth

### CODE REVIEW

File: zh/medium-endswith.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+这个挑战属于中等难度，但我认为它不应该出现在这里。
+它更像一个简单难度的而不是中等难度的。
+但是，我又有什么资格去评判。
+
+我们需要检查字符串是否以特定的字符串结束。
+很明显模板字面量类型会很有用。
+
+让我们从模板字面量类型开始，它可以包含任何字符串。
+在这一点上，我们不关心内容，所以我们在这里使用 `any` 类型：
```
Comment (charleserious) [inline zh/medium-endswith.md:29]: "At this point" in here maybe means "在目前" or "在当前时刻"

**Review by charleserious — COMMENTED** [review]
A tiny little confusion, but everything else LGTM

File: zh/medium-endswith.md (hunk 1 above)
Comment (likui628) [reply zh/medium-endswith.md:29]: I think “目前” more fluently.

**Review by charleserious — APPROVED** [review]
LGTM

**Pull request merged**